### DISCUSSION TITLE
Cant change bit rate of blackhole.

### DISCUSSION THREAD

**Discussion by ryujincoder** [ExistentialAudio/BlackHole#768] [Q&A]
In short i am trying to record Apple Music with blackhole by matching the bit rate and sample rate of the music app and my daw the only problem is is that i cannot change the bit rate of blackhole, it is frozen at 32bit float and i cannot lower it. The sample rate is adjustable but i cannot change the bit rate at all of blackhole. Can someone please explain. Is the bit rate not needed to change or am i missing something?

**Comment by devinroth** [comment]
32bit is completely lossless for 24bit and 16bit. Your DAW should be able to record in whatever bit rate you want regardless of what BlackHole is using.

Devin

**Reply by ryujincoder** [reply]
Wont that alter the quality even if my daw is recording the same bit and sample rate of apple music, after all it is playing through blackhole

**Comment by devinroth** [comment]
Nope.

I won’t go into the details but 99% of music from Apple Music is 16bit int. 32bit float is more than enough to be lossless. 32bit float is enough to be lossless for 24bit int. There are websites out there explaining the math if you’re interested.

Essentially it would be similar to converting a 2k video to 4k video and then converting it back to 2k. The end result will be exactly like the original.

So no, 32bit float will not alter the quality.

Devin
